.github/workflows/reusable-windows-msi.yml

```diff
@@ -17,15 +21,15 @@ env:
 jobs:
   build:
     name: installer for ${{ inputs.arch }}
-    runs-on: windows-latest
+    runs-on: ${{ inputs.os }}
     timeout-minutes: 60
     env:
-      ARCH: ${{ inputs.arch }}
       IncludeFreethreaded: true
     steps:
     - uses: actions/checkout@v4
       with:
         persist-credentials: false
     - name: Build CPython installer
-      run: ./Tools/msi/build.bat --doc -"${ARCH}"
-      shell: bash
+      shell: cmd
```

Comment:
What's the reason for changing the shell? I'd suggest `pwsh` over `cmd` if you want to avoid bash.

Reply:
Just because I had the following error:

```
 ./Tools/msi/build.bat --doc -"${ARCH}"
  Error: bash: command not found
```

So I'm trying different shells.

BTW I moved this PR to a draft as with Windows on Arm runners are still in early testing. I expect lots of trial and error. Don't spend to much time in reviewing (yet!) :)